crates/holochain_zome_types/src/element.rs

```diff
@@ -302,7 +342,7 @@ impl SignedHeaderHashed {
 
 impl From<SignedHeaderHashed> for HeaderHashed {
```

Comment:
This should be possible

```suggestion
impl<T> From<SignedHashed<T>> for HoloHashed<T> {
```